.travis.yml

```
@@ -0,0 +1,35 @@
+---
+language: shell
+env:
+  - IMAGE_NAME="${DOCKER_ORG}/geospaas:latest"
```

It is unlikely to change. Less config in Travis.
```suggestion
  - IMAGE_NAME="nansencenter/geospaas:latest"
```